### ISSUE TITLE
book vs working group repo schism?

### ISSUE THREAD

**Issue by 7yl4r** [ioos/bio_data_guide#94]
I am finding myself confused when just looking at the readme what this repo is for. I think that it has grown to become two separate things: 1) the guide, and 2) a repo for the working group's issues.

I think the SMBD needs a very clean, minimal repo to maximize conversion from confused newcomer into monthly attendee. Can we split these out into two repos?

It would also be nice to have separate issue trackers for each 1 & 2.

**Comment by MathewBiddle** [comment]
I don't necessarily want to create another repository, but I agree with your confusion as I was in the same boat.

Something to note, the [application chapter](https://ioos.github.io/bio_data_guide/applications.html) is built from .Rmd files in the `datasets/` directory. See https://github.com/ioos/bio_data_guide/blob/main/bio-data-guide/02-application.Rmd

My thought was that as folks work through their dataset, the final product would be a .Rmd file that can be added to the book. 

Maybe the "book" becomes a formal collection of examples then? 

Need to think on this a bit.

**Comment by Br-Johnson** [comment]
I tend to agree with Tylar. Might be nice to split this up for clarity. Once the .Rmd files in the datasets folder are mature and the dataset is published the .Rmd files could be brought over manually, unless the child .Rmd path could be a URL.

**Comment by 7yl4r** [comment]
I am reopening this because - although I split things in the README - I think that having the book in the same repo as the working group information is confusing.

**Comment by MathewBiddle** [comment]
move the book to a separate branch and publish from there.

**Comment by MathewBiddle** [comment]
https://docs.github.com/en/pages/getting-started-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site
